Content/Types/FormSelect.php

```diff
@@ -49,6 +50,16 @@ public function __construct(
     /**
      * {@inheritdoc}
      */
+    public function getDefaultParams(PropertyInterface $property = null)
+    {
+        return [
+            'type' => new PropertyParameter('type', 'content'),
```

Comment:
did discuss it with @wachterjohannes we remove the default property and the type need to be set.